pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -83,7 +83,7 @@ python3.pkgs.buildPythonApplication {
     runHook preInstall
 
     make -f misc/Makefile \
-      PYTHON=${python3.pythonOnBuildForHost.interpreter} \
+      PYTHON=${(python3.pythonOnBuildForHost.withPackages (ps: with ps; [ setuptools ])).interpreter} \
```

Comment:
damn I tried that, but at the wrong layer I guess :sweat_smile: :facepalm: 

Good catch